<?xml version="1.0" encoding="UTF-8"?>
<!--
  SPDX-License-Identifier: GPL-2.0-or-later
  SPDX-FileCopyrightText: 2016 Montel Laurent <montel@kde.org>
-->
<ui version="4.0">
 <class>AutoCorrectionWidget</class>
 <widget class="QWidget" name="AutoCorrectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>572</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="enabledAutocorrection">
     <property name="text">
      <string>Enable autocorrection</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Replacement and exception for language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PimCommonAutoCorrection::AutoCorrectionLanguage" name="autocorrectionLanguage"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="exportAutoCorrection">
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importAutoCorrection">
       <property name="text">
        <string>Import...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>469</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ad&amp;vanced Autocorrection</string>
      </attribute>
      <layout class="QGridLayout" name="_8">
       <item row="1" column="1" rowspan="2">
        <layout class="QVBoxLayout" name="_11">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="advancedAutocorrection">
         <property name="text">
          <string>&amp;Enable word replacement</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="_10">
         <item>
          <widget class="QLabel" name="findLabel">
           <property name="text">
            <string>&amp;Find</string>
           </property>
           <property name="buddy">
            <cstring>find</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="find">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="replaceLabel">
           <property name="text">
            <string>Re&amp;place</string>
           </property>
           <property name="buddy">
            <cstring>replace</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="replace">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="PimCommonAutoCorrection::AutoCorrectionTreeWidget" name="treeWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Find</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Replace</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>E&amp;xceptions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Do not treat as the end of a sentence:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="abbreviation">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="2">
          <layout class="QVBoxLayout" name="_14">
           <item>
            <widget class="QPushButton" name="add1">
             <property name="text">
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove1">
             <property name="text">
              <string>R&amp;emove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="PimCommonAutoCorrection::AutoCorrectionListWidget" name="abbreviationList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Accept two uppercase letters in:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="twoUpperLetter">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="2">
          <layout class="QVBoxLayout" name="_16">
           <item>
            <widget class="QPushButton" name="add2">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove2">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="PimCommonAutoCorrection::AutoCorrectionListWidget" name="twoUpperLetterList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Si&amp;mple Autocorrection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="upperCase">
         <property name="whatsThis">
          <string>Detect when a new sentence is started and always ensure that the first character is an uppercase character</string>
         </property>
         <property name="text">
          <string>Convert &amp;first letter of a sentence automatically to uppercase
(e.g. &quot;my house. in this town&quot; to &quot;my house. In this town&quot;)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="upperUpper">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>All words are checked for the common mistake of holding the shift key down a bit too long. If some words must have two uppercase characters, then those exceptions should be added in the 'Exceptions' tab.</string>
         </property>
         <property name="text">
          <string>Convert &amp;two uppercase characters to one uppercase and one lowercase character
 (e.g. PErfect to Perfect)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ignoreDoubleSpace">
         <property name="whatsThis">
          <string>Make sure that more than one space cannot be typed, as this is a common mistake which is quite hard to find in formatted text.</string>
         </property>
         <property name="text">
          <string>&amp;Suppress double spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoReplaceNumber">
         <property name="whatsThis">
          <string>Most standard fraction notations will be converted when available</string>
         </property>
         <property name="text">
          <string>Re&amp;place 1/2... with ½...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="capitalizeDaysName">
         <property name="text">
          <string>Capitalize name of days</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoFormatUrl">
         <property name="whatsThis">
          <string>Detect when a URL (Uniform Resource Locator) is typed and provide formatting that matches the way an Internet browser would show a URL.</string>
         </property>
         <property name="text">
          <string>Autoformat &amp;URLs (HTML mode)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoChangeFormat">
         <property name="whatsThis">
          <string>When you use _underline_ or *bold*, the text between the underscores or asterisks will be converted to underlined or bold text.</string>
         </property>
         <property name="text">
          <string>Automatically do &amp;bold, strikeout and underline formatting (HTML mode)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoSuperScript">
         <property name="text">
          <string>Replace 1st... with 1^st... (HTML mode)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="addNonBreakingSpaceInFrench">
         <property name="text">
          <string>Add non breaking space before specific punctuation marks in French text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="replaceDoubleQuotesByFrenchQuotes">
         <property name="text">
          <string>Replace double quotes by french quotes</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Custom &amp;Quotes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="typographicDoubleQuotes">
         <property name="text">
          <string>Replace &amp;double quotes with typographical quotes</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="2">
            <widget class="QLabel" name="doubleQuoteBeginReplace">
             <property name="text">
              <string>By Default</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="doubleQuote1">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>&quot;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="doubleQuote2">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>&quot;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Ending Quote:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="doubleQuoteEndReplace">
             <property name="text">
              <string>By Default</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3" rowspan="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Opening Quote:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QPushButton" name="doubleDefault">
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="typographicSingleQuotes">
         <property name="text">
          <string>Replace &amp;single quotes with typographical quotes</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="3" rowspan="2">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="singleQuote1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>'</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="singleQuote2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>'</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Opening Quote:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="simpleQuoteBeginReplace">
             <property name="text">
              <string>By Default</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Ending Quotes:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="simpleQuoteEndReplace">
             <property name="text">
              <string>By Default</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QPushButton" name="singleDefault">
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PimCommonAutoCorrection::AutoCorrectionListWidget</class>
   <extends>QListWidget</extends>
   <header>autocorrection/widgets/autocorrectionlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>PimCommonAutoCorrection::AutoCorrectionTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>autocorrection/widgets/autocorrectiontreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>PimCommonAutoCorrection::AutoCorrectionLanguage</class>
   <extends>QComboBox</extends>
   <header>autocorrection/widgets/autocorrectionlanguage.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>enabledAutocorrection</tabstop>
  <tabstop>autocorrectionLanguage</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>upperCase</tabstop>
  <tabstop>upperUpper</tabstop>
  <tabstop>ignoreDoubleSpace</tabstop>
  <tabstop>autoReplaceNumber</tabstop>
  <tabstop>capitalizeDaysName</tabstop>
  <tabstop>autoFormatUrl</tabstop>
  <tabstop>autoChangeFormat</tabstop>
  <tabstop>autoSuperScript</tabstop>
  <tabstop>typographicDoubleQuotes</tabstop>
  <tabstop>doubleQuote1</tabstop>
  <tabstop>typographicSingleQuotes</tabstop>
  <tabstop>singleQuote1</tabstop>
  <tabstop>singleQuote2</tabstop>
  <tabstop>singleDefault</tabstop>
  <tabstop>advancedAutocorrection</tabstop>
  <tabstop>find</tabstop>
  <tabstop>replace</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>abbreviation</tabstop>
  <tabstop>add1</tabstop>
  <tabstop>remove1</tabstop>
  <tabstop>abbreviationList</tabstop>
  <tabstop>twoUpperLetter</tabstop>
  <tabstop>add2</tabstop>
  <tabstop>remove2</tabstop>
  <tabstop>twoUpperLetterList</tabstop>
  <tabstop>exportAutoCorrection</tabstop>
  <tabstop>importAutoCorrection</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
